ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.16.yaml

```diff
@@ -294,6 +294,13 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-gcp-ipi-xpn-cco-manual-users-static
+- as: gcp-ipi-xpn-minimal-permission-byo-zone-f28
+  cron: 31 7 6 * *
+  steps:
+    cluster_profile: gcp-qe
+    test:
+    - ref: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-gcp-ipi-xpn-minimal-permission-byo-zone
```

Comment:
Shared VPC (XPN) with BYO private zone sounds like a feature worth regular E2E regression testing on it, especially for ingress operator, maybe we can consider adding it to our 4.16 CI directly?

Reply:
OK, let me move it to 4.16 CI, thanks!